### PR TITLE
Sundar2021reproducibility

### PR DESCRIPTION
**Pull request by varun19299**
Hi, we got done till step 4, but with a few issues:

* author names don't show up in the [PDF](https://github.com/ReScience/RC-2020/blob/bb647fa78228c2c89c46f4e53559a1fbaeaa3b4d/sundar2021reproducibility/journal/article.pdf)
* Citations, links aren't coloured (should we remove xcolor?)

### CODE REVIEW

**Comment by koustuvsinha** [comment]
Hi @varun19299 , yes please remove `xcolor` :)

**Comment by koustuvsinha** [comment]
@varun19299 can you paste your build.log? I will inspect the author name issue

**Comment by varun19299** [comment]
Would this be [author.log](https://github.com/varun19299/RC-2020/blob/main/sundar2021reproducibility/journal/article.log) or is there another build.log file?

**Comment by koustuvsinha** [comment]
Hi Varun,

It would be the output of make command. To capture it, run “make > build.log”.

Thanks,

Koustuv

On Apr 9, 2021, at 11:42 PM, Varun Sundar ***@***.***> wrote:

﻿

Would this be author.log<https://github.com/varun19299/RC-2020/blob/main/sundar2021reproducibility/journal/article.log> or is there another build.log file?

**Comment by varun19299** [comment]
Fixed author names by setting abstract as empty string in `metadata.yaml`

**Comment by varun19299** [comment]
Also resized a couple of tables. We might reshape another figure to better suit the layout.

**Comment by varun19299** [comment]
Here's the [build.log](https://gist.github.com/varun19299/5da1e8c8d4159c408454c43ca8b58382).

File: sundar2021reproducibility/openreview/packages.tex [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+\usepackage{multirow}
+\usepackage{comment}
+\usepackage{makecell}
+\usepackage{amsfonts}       % blackboard math symbols
+\usepackage{amsmath}
+\usepackage{nicefrac}       % compact symbols for 1/2, etc.
+\usepackage{microtype}      % microtypography
+\usepackage{graphicx}      % images
+\usepackage[english]{babel}
```
Comment (koustuvsinha) [inline sundar2021reproducibility/openreview/packages.tex:40]: It seems this import is the issue from the logs: "! LaTeX Error: Option clash for package babel." Can you disable this and re-build?

File: sundar2021reproducibility/openreview/packages.tex [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+%     \usepackage[preprint]{neurips_2019}
+
+% to compile a camera-ready version, add the [final] option, e.g.:
+%\usepackage[final]{neurips_2019} % Remove for ReScience
+
+% to avoid loading the natbib package, add option nonatbib:
+% \usepackage[nonatbib, final]{neurips_2019}
+
+\usepackage[utf8]{inputenc} % allow utf-8 input
+\usepackage[T1]{fontenc}    % use 8-bit T1 fonts
```
Comment (koustuvsinha) [inline sundar2021reproducibility/openreview/packages.tex:20]: Please remove this import `fontenc` as this appears to break ReScience fonts (check #4 for discussions)

**Comment by koustuvsinha** [comment]
> Can citation links be coloured?

Unfortunately, it seems rescience.cls does not support colored citation links.

**Comment by varun19299** [comment]
Removed these packages. Updated [PDF](https://github.com/ReScience/RC-2020/blob/edfa0a92b2f260977e27941c725dbe67f2d0808b/sundar2021reproducibility/journal/article.pdf).

Any suggestions for fixing the inline math overflow on page 9? (\alpha, \Delta T =...)

File: sundar2021reproducibility/openreview/packages.tex [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+\usepackage{amsfonts}       % blackboard math symbols
+\usepackage{amsmath}
+\usepackage{nicefrac}       % compact symbols for 1/2, etc.
+\usepackage{microtype}      % microtypography
+\usepackage{graphicx}      % images
+
+% Disable for Rescience C
+% \usepackage[english]{babel}
+
+\bibliographystyle{plainnat}
```
Comment (koustuvsinha) [inline sundar2021reproducibility/openreview/packages.tex:47]: Disable this import, seems to break compilation on my end

File: sundar2021reproducibility/openreview/sections/methodology.tex [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+
+\subsubsection{Mask Initialization}}For a network with $L$ layers and total parameters $N$, we associate each layer with a random boolean mask of sparsity $s_l, \; l \in [L]$. The overall sparsity of the network is given by $S=\frac{\sum_l s_l N_l}{N}$, where $N_l$ is the parameter count of layer $l$. Sparsities $s_l$ are determined by the one of the following mask initialization strategies:
```
Comment (koustuvsinha) [inline sundar2021reproducibility/openreview/sections/methodology.tex:6]: There is an extra `}` which is causing latex overflow of equation in page 9

Comment (varun19299) [reply sundar2021reproducibility/openreview/sections/methodology.tex]: Fixed this in [commit](https://github.com/ReScience/RC-2020/pull/2/commits/d8f404ed4db5a7deccdb844d0d6ff6369b1f6255).

File: sundar2021reproducibility/openreview/packages.tex (hunk 3 above)
Comment (varun19299) [reply sundar2021reproducibility/openreview/packages.tex]: Done in [commit](https://github.com/ReScience/RC-2020/pull/2/commits/d8f404ed4db5a7deccdb844d0d6ff6369b1f6255).

**Comment by koustuvsinha** [comment]
Thanks for the changes! LGTM

**Comment by varun19299** [comment]
Hi @koustuvsinha, we're done making all changes from our end. Please let us know if anything else is pending.

**Comment by koustuvsinha** [comment]
LGTM on the latex side. @dodgejesse can you review the content side? Then we will be good to go.

**Review by dodgejesse — APPROVED** [review]
Looks good to me. The authors could optionally remove the whitespace from the page break on the second page (after the summary, before the intro), but that's just an aesthetic choice. It looks like an appendix has also been added, which is fine. Good work!

**Pull request merged**